sdk/python/feast/infra/aws.py

```diff
@@ -38,11 +39,14 @@
 
     raise FeastExtrasDependencyImportError("aws", str(e))
 
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+_handler = logging.StreamHandler()
+_handler.setLevel(logging.INFO)
+_logger.addHandler(_handler)
 
```

Comment:
You shouldn't need to do any of the `setLevel` or `addHandler` business here - you only need to get a logger using `getLogger` and you can use it out of the box.

Any configuration should be happening in `feast/__init__.py`, or potentially using a logging file.